### ISSUE TITLE
wxpython 3.0 with python 2.7 

### ISSUE THREAD

**Issue by tolondra** [Nuitka/Nuitka#2201]
Dear Nuitka developers, 

We are trying to build an executable for an application that uses wxpython 3.0 and python 2.7.

When "compiled" with: 'python -m nuitka example2.py --standalone'

        wxpython 3.0
        Python 2.7.15
        Nuitka 0.6.16
        on Microsoft Windows 11 Education, Version 10.0.22621 Build 22621

    It leads to an exe that starts but crashes as soon as a tab is added to the notebook.

    (with python 3.10.x and an up-to-date nuitka it works)

Attached a source code example and the executable (that crashes).

Is there any solution for Python 2.7? 

Cheers,

Natalia

**Comment by kayhayen** [comment]
Hello Natalia, we do not execute binaries we didn't compile outselves.

**Comment by kayhayen** [comment]
In the template, I have made it clear, that not filling out the issue template is not acceptable. I am closing this kind of report usually as invalid. The main reason is that I do not want to be left guessing. For you it's easy to erase the issue template and expect me to ask you questions only as necessary, but I cannot do that. I would much rather wait for you to create a new report, fill out the template, or wait for somebody with the necessary skill to do it.

In short: No issue template, no fun. 

I do not mind it being about python2.7, but I do mind missing the minimal example. If you cannot, let me know and I will delete the report.

**Comment by tolondra** [comment]
Hi Kayhaye,

Thanks a lot for your message. I am very sorry I did not follow the template. I oversee it. I will remove the issue and post a new one!

Cheers,

Natalia

 BTW, this was not necessary:  " wait for somebody with the necessary skill to do it.

**Comment by kayhayen** [comment]
@tolondra it's a template text, I cannot refine it per use, it's bad enough how often I have to use it
